proxy/app.js

```diff
@@ -12,7 +12,7 @@ const items = {
     }
   },
   address: {
-    target: 'https://api.vic.gov.au:443/delwp/address/v1/suggestions', // target host
+    target: 'https://wovg-shared.gateway.prod.api.vic.gov.au/delwp/address/v1/suggestions', // target host
```

Comment:
Hi @anthony-malkoun this proxy for address has been replaced by another better one Nick created for us. See https://github.com/dpc-sdp/myvic-components/blob/develop/packages/AddressSearch/utils/getAddresses.js#L10
It's now: https://api.sdp.vic.gov.au/delwp/address/v1/suggestions?query=
You may need to do a update there instead of here.